rasa/shared/exceptions.py

```diff
@@ -17,6 +18,10 @@ class InvalidParameterException(RasaException, ValueError):
     """Raised when an invalid parameter is used."""
 
 
+class MarkdownException(RasaException, ValueError):
```

Comment:
Why is this a `RasaException`?

Reply:
it's an exception generated by user input which is expected, checkout the PR desc:
> Raise MarkdownException when Markdown cannot be read ([sentry/1953580685](https://sentry.io/organizations/rasahq/issues/1953580685/?project=2801673))

Reply:
Ah ok - I interpreted "Markdown cannot be read" as meaning invalid markdown, rather than not fitting our spec.